src/Terminus.php

```diff
@@ -1,58 +1,81 @@
 <?php
 // src/Terminus.php
 
+namespace Pantheon;
+
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+use Symfony\Component\ClassLoader\Psr4ClassLoader;
 use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
 use Terminus\Config;
 
-class Terminus extends Application {
+class Terminus extends Application
+{
+  /**
+   * @var Config
+   */
+  private $config;
 
   /**
    * @inheritdoc
    */
-  public function __construct() {
-    $this->setAutoloader();
-    parent::__construct('Terminus', Config::get('version'));
-    date_default_timezone_set(Config::get('time_zone'));
-  }
+    public function __construct()
+    {
+        $this->config = new Config();
+        parent::__construct('Terminus', $this->config->get('version'));
+        date_default_timezone_set($this->config->get('time_zone'));
```

Comment:
Not sure you want to do this unconditionally. What if the timezone isn't set in Terminus' config, but is set in ini.php?